src/experiment_prototype/experiment_slice.py

```diff
@@ -620,6 +620,15 @@ def check_freq(cls, freq):
 
         return freq
 
+    @validator('decimation_scheme')
+    def check_decimation_rates(cls, decimation_scheme, values):
+        input_rate = decimation_scheme.input_rates[0]
```

Comment:
Should there first be a check to ensure that `input_rates` is actually a property of decimation_scheme? Or is this done by the `decimation_scheme: DecimationScheme = Field(default_factory=create_default_scheme)` definition?

Reply:
It's defined in the `__init__` of the DecimationScheme object, so if for some reason it isn't a property then it will fail hard anyways.